templates/portfolios/detail.html

```diff
@@ -31,12 +31,11 @@
     <div class="portfolio-top">
     <div class="container">
     <div class="portfolio-main">
-        <a id="new-project" onclick="show_select_portfolio_modal(); return false;" href="#" class=" btn
-        btn-success">Start a project</a>
         {% if can_change_portfolio %}
-            <a id="manage-portfolio" onclick="show_manage_portfolio_modal('{{ portfolio.id }}', '{{ portfolio.title }}'); return false;" href="#"
-               class=" btn btn-success">Manage Portfolio</a>
+            <a id="manage-portfolio" href="/portfolios/{{ portfolio.id }}/edit" class=" btn">Edit Portfolio</a>
```

Comment:
The "Django doctor" has started flagging hardcoded urls in our href attributes. To reduce technical debt, and ensure our hrefs are dynamic (in case the path changes in one place, we want the change to propagate everywhere), we can structure our paths using the "url" function, like this:
`href="{% url '<name_of_portfolio_edit_url>' portfolio.id %}"`